portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,46 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html, body {
+  height: 100%; 
+  margin: 0px;
+  font-family: sans-serif;
```

Comment:
In a future pull request I'd try tinkering around with different [Google Fonts](https://fonts.google.com/)

Font choice is probably one of the biggest impact design decisions you can make for your website.